### ISSUE TITLE
scylla_setup failed to setup RAID on Ubuntu 20.04

### ISSUE THREAD

**Issue by enaydanov** [scylladb/scylla#7627]
`scylla_setup` unable to setup RAID because Ubuntu 20.04 used `/dev/md0` already.

Both checks for existence of `/dev/md0` (4.3) and `/sys/block/md0/md/array_state` (master) failed.

Tested on AWS and GCE, both w/ and w/o NVMe's

*Installation details*

Scylla version (or git commit hash): 4.3.rc1-0.20201115.b2271800a55 and 4.4.dev-0.20201113.0a2adf4555
AWS: ami-0fb4089e765e1ea02 (eu-north-1) [Official Ubuntu 20.04 LTS AMI]
GCE:  https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts

```
$ sudo /usr/lib/scylla/scylla_setup --nic ens5 --disks /dev/nvme0n1 --setup-nic-and-disks  --swap-directory / 
...
/dev/md0 is already using
RAID setup failed.
```

#6343

**Comment by tzach** [comment]
@enaydanov is this issue related on Ubuntu 20? does the same test pass for Ubuntu 18?

**Comment by syuu1228** [comment]
I think we can close this since we merged 587b909c5ce35c74458650faffdc1fa464a338b9

**Comment by avikivity** [comment]
@enaydanov please verify.

**Comment by enaydanov** [comment]
> @enaydanov is this issue related on Ubuntu 20? does the same test pass for Ubuntu 18?

Yes, it's related to Ubuntu 20 only.

**Comment by enaydanov** [comment]
Artifacts test passed: https://jenkins.scylladb.com/view/master/job/scylla-master/job/artifacts/job/artifacts-ubuntu2004-test/10/

ScyllaDB version: 4.4.dev-0.20201130.ea9c058be3
ScyllaDB repo: http://downloads.scylladb.com/deb/unstable/unified/master/2020-11-30T15:41:07Z/scylladb-master/scylla.list
Actual gce image used: ubuntu-2004-focal-v20201111
AWS region: us-east1
Instance type: n1-standard-2

**Comment by avikivity** [comment]
Fixed by 587b909c5ce35c74458650faffdc1fa464a338b9.

**Comment by amoskong** [comment]
@avikivity @tzach we need to backport this fix, if we want scylla 4.3 work with Ubuntu 20.04.

**Comment by avikivity** [comment]
Backported to 4.3.
